core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -473,6 +474,12 @@ angular.module('oppia').directive('questionsList', [
                     }
                     $uibModalInstance.close();
                   };
+                  // Checking if Question contains all requirement to enable
+                  // Save and Publish Question
+                  $scope.toggleEnableButton = function() {
```

Comment:
Rename the function to something like `isSaveButtonDisabled()`.

Reply:
ok :)